### ISSUE TITLE
favicon

### ISSUE THREAD

**Issue by reynoldsnlp** [bigskysoftware/htmx#2300]
I already submitted a PR for this (#2294), but I probably should have made an issue first. I think it would be nice to have a favicon for the htmx.org website. I could submit a similar PR for hypermedia.systems, too, if that would be interesting.

**Comment by timkofu** [comment]
> I think it would be nice to have a favicon for the htmx.org website.

Yes. When the tab is pinned, it's that wiry globe up there. Doesn't look right.

**Comment by mehedi-etlimited** [comment]
> I already submitted a PR for this (#2294), but I probably should have made an issue first. I think it would be nice to have a favicon for the htmx.org website. I could submit a similar PR for hypermedia.systems, too, if that would be interesting.

without the favicon it looks incomplete

**Comment by 1cg** [comment]
fixed
